### PR TITLE
Docs for Desktop 3.3.1 release

### PR DESCRIPTION
**Pull request by usha-mandya**
Docs updates for Docker Desktop 3.3.1 release

### CODE REVIEW

File: docker-for-mac/apple-silicon.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+Click the following link to download the GA version of Docker Desktop for Apple silicon.
+
+> [Download](https://desktop.docker.com/mac/stable/arm64/Docker.dmg)
+
+### System requirements
+
+You must install **Rosetta 2** as some binaries are still Darwin/AMD64. To install Rosetta 2 manually from the command line, run the following command:
+
+    ```
```
Comment (StefanScherer) [inline docker-for-mac/apple-silicon.md:25]: Somehow the formatting doesn't work, the three backticks are visible: https://deploy-preview-12663--docsdocker.netlify.app/docker-for-mac/apple-silicon/#system-requirements

Comment (StefanScherer) [inline docker-for-mac/apple-silicon.md:25]: Maybe just indent the command without backticks and move the next sentence to the left.

```suggestion
    softwareupdate --install-rosetta

We expect to fix this in a future release.
```

Comment (usha-mandya) [reply docker-for-mac/apple-silicon.md:25]: Thanks for spotting this. I think it's the tab. Fixed this on my local branch. Will merge with a couple of other changes

File: docker-for-mac/apple-silicon.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+description: Docker Desktop for Apple silicon
+keywords: Docker Desktop, M1, Silicon, Apple,
+title: Docker Desktop for Apple silicon
+toc_min: 2
+toc_max: 3
+redirect_from:
+- /docker-for-mac/apple-m1/
+---
+
+Welcome to Docker Desktop for Apple Silicon.
```
Comment (StefanScherer) [inline docker-for-mac/apple-silicon.md:11]:
```suggestion
Welcome to Docker Desktop for Apple silicon.
```

File: docker-for-mac/install.md [hunk 3]
```diff
@@ -87,6 +102,7 @@ To uninstall Docker Desktop from your Mac:
 ## Where to go next
 
 - [Getting started](index.md) provides an overview of Docker Desktop on Mac, basic Docker command examples, how to get help or give feedback, and links to other topics about Docker Desktop on Mac.
+- [Docker Desktop for Apple silicon](apple-silicon.md)for detailed information about Docker Desktop for Apple silicon.
```
Comment (StefanScherer) [inline docker-for-mac/install.md:105]:
```suggestion
- [Docker Desktop for Apple silicon](apple-silicon.md) for detailed information about Docker Desktop for Apple silicon.
```

File: docker-for-mac/release-notes.md [hunk 4]
```diff
@@ -11,10 +11,25 @@ toc_max: 2
...
+## Docker Desktop 3.3.1
+2021-04-15
+
+> [Download for Mac with Intel chip](https://desktop.docker.com/mac/stable/amd64/Docker.dmg)
+
+> [Download for Mac with Apple chip](https://desktop.docker.com/mac/stable/arm64/Docker.dmg)
+
+### New
+
+Docker Desktop for Mac on Apple silicon is now available a GA release. For more information, see [Docker Desktop for Apple silicon](apple-silicon.md).
```
Comment (StefanScherer) [inline docker-for-mac/release-notes.md:23]: a -> as ?

File: docker-for-mac/apple-silicon.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+   However, attempts to run Intel-based containers on Apple Silicon machines can crash as QEMU sometimes fails to run the container. Filesystem change notification APIs (e.g. `inotify`) do not work under QEMU emulation, see [docker/for-mac#5321](https://github.com/docker/for-mac/issues/5321). Therefore, we recommend that you run ARM64 containers on Apple Silicon machines. These containers are also faster and use less memory than Intel-based containers.
+
+   We expect this issue to become less common over time, as more and more images are rebuilt [supporting multiple architectures](https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/).
+
+- When using the `qemu` backend, `ping` from inside a container to the Internet does not work as expected. To test the network, we recommend using `curl` or `wget`. See [docker/for-mac#5322](https://github.com/docker/for-mac/issues/5322#issuecomment-809392861).
```
Comment (djs55) [inline docker-for-mac/apple-silicon.md:39]: We've fixed this bug, so the known issue can be removed.

File: docker-for-mac/apple-silicon.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+### Fixes since Docker Desktop RC 3
+
+- Docker Desktop now ensures the permissions of `/dev/null` are correctly set to `0666` (`rw-rw-rw-`) inside `--privileged` containers. Fixes [docker/for-mac#5527](https://github.com/docker/for-mac/issues/5527).
+- Fixed an issue where ICMP echo responses (`ping` responses) had incorrect sequence numbers.
```
Comment (djs55) [inline docker-for-mac/apple-silicon.md:44]: For `ping` I think we can reference https://github.com/docker/for-mac/issues/5322#issuecomment-809392861

**Review by stephen-turner — CHANGES_REQUESTED** [review]
Looks good, Usha. I like the new tabs.

I was a bit worried about the duplicated release notes on Mac, but I think it works for now, although we should remove them from the M1 page at the next release.

The only change we should make is to add one more exception to the support statement (both Mac and Windows) as follows:
* Running containers of a different architecture using emulation

File: docker-for-mac/apple-silicon.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+- /docker-for-mac/apple-m1/
+---
+
+Welcome to Docker Desktop for Apple silicon.
+
+## Docker Desktop 3.3.1
+
+2021-04-15
+
+Click on the following link to download the GA version of Docker Desktop for Apple silicon.
```
[the remark this answers is not in the document]
Comment (usha-mandya) [reply docker-for-mac/apple-silicon.md]: The title emphasizes Apple silicon, so decided simplify this statement and updated it to 'Click the following link to download Docker Desktop'.

File: docker-for-mac/install.md [hunk 8]
```diff
@@ -66,7 +94,7 @@ and select **Learn**.
 
 Starting with Docker Desktop 3.0.0, updates to Docker Desktop will be available automatically as delta updates from the previous version.
 
-When an update is available, Docker Desktop displays an icon to indicate the availability of a newer version. You can start downloading the update in the background whenever it is convenient for you.
+When an update is available, Docker Desktop displays an icon to indicate the availability of a newer version. You can start downloading the update in the background whenever it is convenient for you. Alternatively, you can click **Snooze** to pause the update.
```
[the remark this answers is not in the document]
Comment (usha-mandya) [reply docker-for-mac/install.md]: That's a good point. Snooze is a fairly simple option and doesn't need a description on how to use. I've removed this statement.

**Review by StefanScherer — DISMISSED** [review]
LGTM 🎉

File: docker-for-mac/apple-silicon.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+
+   We expect this issue to become less common over time, as more and more images are rebuilt [supporting multiple architectures](https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/).
+
+- When using the `qemu` backend, `ping` from inside a container to the Internet does not work as expected.  To test the network, we recommend using `curl` or `wget`. See [docker/for-mac#5322](https://github.com/docker/for-mac/issues/5322#issuecomment-809392861).
+
+### Fixes since Docker Desktop RC 3
+
+- Docker Desktop now ensures the permissions of `/dev/null` and other devices are correctly set to `0666` (`rw-rw-rw-`) inside `--privileged` containers. Fixes [docker/for-mac#5527](https://github.com/docker/for-mac/issues/5527).
+- Fixed an issue where ICMP echo responses (`ping` responses) had incorrect sequence numbers.
+- Fixed an issue where data was dropped when a TCP stream was half-closed.
```
Comment (djs55) [inline docker-for-mac/apple-silicon.md:49]: Unfortunately these 2 fixes (ICMP echo responses and TCP stream half-closed) got reverted: (when we switched back from `vpnkit` networking to `qemu` networking to fix wake from sleep today)

**Review by stephen-turner — APPROVED** [review]
Ship it!

**Pull request merged**